src/chainparams.cpp

```diff
@@ -540,6 +546,12 @@ class CTestNetParams : public CChainParams {
                 Consensus::FS_ZIP214_MG,
                 consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight, 2796000,
                 mg_addresses);
+
+            //consensus.AddZIP207LockboxStream(
+            //    keyConstants,
+            //    Consensus::FS_ZIPTBD_LB,
```

Comment:
Let's avoid inscrutible acronyms.
```suggestion
            //    Consensus::FS_ZIPTBD_LOCKBOX,
```